GOVERNANCE.md

```diff
@@ -64,10 +64,6 @@ A guide for Collaborators is maintained in
 ## CTC Membership
 
 CTC seats are not time-limited.  There is no fixed size of the CTC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
```

Comment:
@Fishrock123 If others agree strongly with that, then sure. I took it out for three reasons, from least important to most important:

* All other things being equal, less text is better than more. 

* I feel like what's stated there is a given and so doesn't need to be stated explicitly. Of *course* we're looking to cover all important areas of expertise. Of *course* we want to be able to operate efficiently.

* I also want to avoid the possible misunderstanding that things listed there are the *only* factors to be considered. Let's imagine the CTC were hypothetically composed of only Americans in San Francisco, Austin, and New York. Even if all areas of expertise were covered, and even if it hampers operational efficiency, it would (in this purely hypothetical situation) be worthwhile to seek out voices from other parts of the world. But that's not covered (or at least it isn't if you infer "important areas of expertise" to be strictly technical, which I suspect most readers do).

So those were my reasons, but again, if others feel strongly that it ought to remain, then that's fine. There are other considerations and that one sentence of text is not something I want to have a prolonged discussion about, certainly not at this time when all I'm really trying to do is fix the number of members part.